### PR TITLE
Refactor Form CRUD UX

### PR DESCRIPTION
**Pull request by sergei-maertens**
* Introduced tabs to manage various aspects of a form
* Moved some fields around into a dedicated tab
* Set up react-intl and translation machinery, see docs

@shea-maykinmedia I've moved the registration backend options (together with backend selection) into its own "Registratie" tab

### CODE REVIEW

File: src/openforms/js/components/admin/form_design/RegistrationFields.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+const RegistrationFields = ({
+    backends=[],
+    selectedBackend='',
+    backendOptions={},
+    onChange,
+}) => {
+    const backendChoices = backends.map( backend => [backend.id, backend.label]);
+
+    // TODO: proper UI, this doesn't work
```
Comment (shea-maykinmedia) [inline src/openforms/js/components/admin/form_design/RegistrationFields.js:19]: Is this something I should do as part of #547 ?

**Review by shea-maykinmedia — APPROVED** [review]

**Review by SilviaAmAm — COMMENTED** [review]
Looks good! Just tiny comments

File: src/openforms/js/components/admin/form_design/RegistrationFields.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        const fakeEvent = {target: {name, value: parsedValue}};
+        onChange(fakeEvent);
+    };
+
+    return (
+        <Fieldset>
+            <FormRow>
+                <Field
+                    name="form.registrationBackend"
+                    label="Select registration backend"
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/js/components/admin/form_design/RegistrationFields.js:38]: Yes - `node` is anything that can be rendered (component, string, number...)! So translation is still pending, but just a string works as well.

File: src/openforms/js/components/admin/form_design/RegistrationFields.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                        value={selectedBackend}
+                        onChange={onChange}
+                        allowBlank={true}
+                    />
+                </Field>
+            </FormRow>
+            <FormRow>
+                <Field
+                    name="form.registrationBackendOptions"
+                    label="Registration Backend Options"
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/js/components/admin/form_design/RegistrationFields.js:52]: Yes - `node` is anything that can be rendered (component, string, number...)! So translation is still pending, but just a string works as well.

[the remark this answers is not in the document]
Comment (SilviaAmAm) [reply src/openforms/js/components/admin/form_design/RegistrationFields.js]: Ohh! I see :)

**Pull request merged**